.github/helm-docs.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-HELM_DOCS_VERSION="1.9.1"
+HELM_DOCS_VERSION="1.11.0"
```

Comment:
The last version fix 
fix : https://github.com/ManagedKube/helm-charts/actions/runs/3147974652/jobs/5118020452#step:3:32